doc/reference/reference_lua/box_ctl.rst

```diff
@@ -44,6 +44,12 @@ Below is a list of all ``box.ctl`` functions.
         *  - :doc:`./box_ctl/on_shutdown`
            - Create a "shutdown trigger"
 
+        *  - :doc:`./box_ctl/on_recovery_state`
+           - Create a trigger executed on different stages of a node recovery or initial configuration
+
+        *  - :doc:`./box_ctl/on_election`
+           - Create a :ref:`trigger <triggers>` executed every time the current state of a replica set node in regards to :ref:`leader election <repl_leader_elect>` changes
```

Comment:
```suggestion
           - Create a :ref:`trigger <triggers>` executed every time the current state of a replica set node in regard to :ref:`leader election <repl_leader_elect>` changes
```
The phrase "in regards" (with -s) is considered a nonstandard phrase (see [this
 LanguageTool post](https://languagetool.org/insights/post/regard/#:~:text=The%20phrase%20in%20regard%20to,is%20considered%20a%20nonstandard%20phrase)).
 
Actually, this sentence looks a bit complex and wordy to me. It might be difficult to understand because the subject and verb "changes" are separated by a large number of words.